Gopkg.toml

```diff
@@ -18,10 +18,15 @@
   name = "github.com/mattes/migrate"
   revision = "ce1b59bb819416ca67422929b1cb48bb342720b6"
 
+[[constraint]]
```

Comment:
Where did this come from?

Reply:
The override was added in [this commit](https://github.com/weaveworks/flux/commit/8d84d8d4b55ec84f399d808a72a89750414b8039), presumably because of a build issue, but I've just removed the constraint and the repo still builds. I changed it to a constraint instead of an override just as a cleanup, but I can go one better and just remove it altogether.